automation/src/test/java/com/microsoft/identity/common/test/automation/AcquireTokenAuthorityTest.java

```diff
@@ -121,12 +125,37 @@ private static User getUser(TestConfigurationQuery query) {
         return newUser;
     }
 
+    @Test
+    public void should_be_able_to_acquire_access_token_with_validate_authority_false_for_valid_authority(){
+        givenThat(james).wasAbleTo(
+                clearCache,
+                clickDone,
+                acquireToken.validateAuthority(false),
+                clickDone,
+                readCache);
+        int expectedCacheCount = james.asksFor(ExpectedCacheItemCount.displayed());
+        then(james).should(seeThat(TokenCacheItemCount.displayed(), is(expectedCacheCount)));
+    }
+
+    // invalid authority test, no validation but should exit gracefully
+    @Test
+    public void should_be_able_to_exit_silently_on_validate_authority_false_for_invalid_authority(){
+        james.attemptsTo(
+                clearCache,
+                clickDone,
+                acquireToken
+                        .validateAuthority(false)
+                        .withAuthority("https://www.cnn.com/world/"));
```

Comment:
We should change this value to use `contoso` or another Microsoft-owned domain value